instaclustr/provider.go

```diff
@@ -23,13 +23,14 @@ func Provider() *schema.Provider {
 		},
 
 		ResourcesMap: map[string]*schema.Resource{
-			"instaclustr_cluster":        resourceCluster(),
-			"instaclustr_encryption_key": resourceEncryptionKey(),
-			"instaclustr_firewall_rule":  resourceFirewallRule(),
-			"instaclustr_vpc_peering":    resourceVpcPeering(),
-			"instaclustr_kafka_user":     resourceKafkaUser(),
-			"instaclustr_kafka_topic":    resourceKafkaTopic(),
-			"instaclustr_kafka_acl":      resourceKafkaAcl(),
+			"instaclustr_cluster":         resourceCluster(),
+			"instaclustr_encryption_key":  resourceEncryptionKey(),
+			"instaclustr_firewall_rule":   resourceFirewallRule(),
+			"instaclustr_vpc_peering_aws": resourceVpcPeering(),
+			"instaclustr_vpc_peering_gcp": resourceGCPVpcPeering(),
```

Comment:
**[Question, State Upgrade]** Renaming the resource mapping to delineate between GCP and AWS might cause issues for existing AWS VPCs. Has that been considered? It seems like something that can be fixed for customers with existing VPC declarations with `terraform mv instaclustr_vpc_peering.<NAME> instaclustr_vpc_peering_aws.<NAME>` but you'd need to confirm that.

Reply:
Yup, thats right I will change the existing AWS resource name back to instaclustr_vpc_peering.go